src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+const Header = ({ title, subtitle }) => (
+  <section id="header">
+    <h1>{ title }</h1>
+    <h2>{ subtitle }</h2>
+  </section>
+);
```

Comment:
Nice job so far 👏, I would suggest avoiding unnecessary complexity.  You can add the title and subtitle directly in the JSX code.

Reply:
Hello @Gh-Reza Thank you very much for your review. I have taken your suggestions into account and have applied the suggested changes. You can see the changes in the last commits